Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -207,19 +207,27 @@ - (void)loadInterstitialAdForParameters:(id<MAAdapterResponseParameters>)paramet
     
     if ( ![VungleAds isInitialized] )
     {
-        [self log: @"Vungle SDK not successfully initialized: failing interstitial ad load..."];
-        [delegate didFailToLoadInterstitialAdWithError: MAAdapterError.notInitialized];
+        [self initializeWithParameters:(id<MAAdapterInitializationParameters>)parameters completionHandler:^(MAAdapterInitializationStatus initializationStatus, NSString * _Nullable errorMessage) {
```

Comment:
This implementation doesn't follow AdMob implementation. If you look at `initializeWithParameters`, it looks like it's only going to attempt initializing once. If it fails, then it won't try again.